test/clojure/core_test/format.cljc

```diff
@@ -1,8 +1,9 @@
+#_{:clj-kondo/ignore [:unused-referred-var]}
```

Comment:
This isn't something we should have to put in every file which isn't using a referred var. If we want kondo to be clean, we should disable these diagnostics at a project level. Same goes for things like inline defs and unused values.